packages/widget-reorder/src/widget.tsx

```diff
@@ -41,6 +41,7 @@ export const createWidget = ({
     const collection: string = field.get('collection')
     const fieldId: string = field.get('id_field')
     const displayFields: List<string> = field.get('display_fields')
+    const maxHeight: string = field.get('max_height') || 'none'
```

Comment:
We could replace `field.get('max_height') || 'none'` to `field.get('max_height', 'none')`

Reply:
ohh nice! Will edit this in